### PR TITLE
✨  Expose WebClient TTS config

### PR DESCRIPTION
**Pull request by marcher357**
## Proposed Changes

Expose rate and pitch from speechsynthesis.ts and add resetsessiononrepromptlimit property to repromptprocessor.ts. 

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project
- [x] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

**Comment by rmtuckerphx** [comment]
@aswetlow @jankoenig I've been working with my co-worker on this PR.
Can you please take a look?

**Review by jankoenig — APPROVED** [review]
Looks great. Thank you @marcher357 and @rmtuckerphx 🎉

File: clients/client-web/src/standalone/SpeechSynthesizer.ts [hunk 1]
```diff
@@ -110,6 +112,12 @@ export class SpeechSynthesizer extends TypedEventEmitter<SpeechSynthesizerEventL
       if (this.config.voice) {
         utterance.voice = this.config.voice;
       }
+      if (this.config.rate) {
+        utterance.rate = this.config.rate;
+      }
+      if (this.config.pitch) {
```
Comment (aswetlow) [inline clients/client-web/src/standalone/SpeechSynthesizer.ts]: According to the docs (https://developer.mozilla.org/en-US/docs/Web/API/SpeechSynthesisUtterance/pitch#value)
pitch = 0 is a valid value. 

`this.config.pitch` = 0 wouldn't be set. I would recommend checking for not undefined

**Review by aswetlow — CHANGES_REQUESTED** [review]

**Comment by aswetlow** [comment]
Thank you for the PR @marcher357 and @rmtuckerphx

File: clients/client-web/src/standalone/SpeechSynthesizer.ts (hunk 1 above)
Comment (marcher357) [reply clients/client-web/src/standalone/SpeechSynthesizer.ts]: done

Comment (aswetlow) [reply clients/client-web/src/standalone/SpeechSynthesizer.ts]: thank you

**Review by aswetlow — APPROVED** [review]

**Pull request merged**